sdk/src/main/java/org/coralibre/android/sdk/internal/datatypes/IntervalOfCapturedData.java

```diff
@@ -0,0 +1,13 @@
+package org.coralibre.android.sdk.internal.datatypes;
+
+import java.util.List;
+
+public interface IntervalOfCapturedData {
```

Comment:
Maybe this interface can be removed? It has (and will always have) only one implementation